CHANGELOG.md

```diff
@@ -44,7 +44,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
   * LBRY URLs not working on Linux ([#1120](https://github.com/lbryio/lbry-app/issues/1120))
   * Fix Windows notifications not showing ([#1145](https://github.com/lbryio/lbry-app/pull/1145))
   * Fix export issues ([#1163](https://github.com/lbryio/lbry-app/pull/1163))
-
+  * Fix Keyboard Shortcut to Quit APP ([#1163](https://github.com/lbryio/lbry-app/pull/1202))
```

Comment:
Please rename this as "Add keyboard shortcut to quit the app on Windows" under an unreleased section and not under "Fix".

Reply:
@IGassmann i cannot see where a unreleased section

Reply:
You have to add one.

Reply:
https://github.com/lbryio/lbry-app/pull/1202/commits/572d68d880d2d216d74df66662600e59d439a54e done

Reply:
It also needs to be under an Added subsection. Also, notice that the pull request number doesn't match the link number.